sites/geohub/src/routes/api/stac/[id]/[collection]/[...item]/tools/[tool]/+server.ts

```diff
@@ -0,0 +1,14 @@
+import type { RequestHandler } from '@sveltejs/kit';
+import { createDatasetLinks } from '$lib/server/helpers';
+import { env } from '$env/dynamic/private';
+
+export const POST: RequestHandler = async ({ request, url }) => {
+	const feature = await request.json();
+
+	// const tool = params.tool;
+
+	feature.properties = await createDatasetLinks(feature, url.origin, env.TITILER_ENDPOINT);
+
+	console.log(feature);
+	return new Response(JSON.stringify(feature));
+};
```

Comment:
looks like this API is not used. remove this.

Reply:
I have used it in the `getToolsFeature` function of `StacApiExplorer` component